schema_utils.py

```diff
@@ -55,8 +55,13 @@
 def normalize_against_schema(obj, schema, apply_custom_validators=True):
     """Validate the given object using the schema, normalizing if necessary.
 
+    Args:
+        obj: *. The object to validate and normalize.
+        schema: str. The schema to validate and normalize the value against.
```

Comment:
Hang on -- why is this a string? /cc @vibhor98

Reply:
@seanlip @vibhor98 It is not str?

Reply:
@DiegoCaraballo, the schema is in the dict format containing key-value pairs. You can actually get the type using `type(schema)`. In general, this schema is for the objects defined in `extensions/objects/models/objects.py`. You can have a look there for reference.

Reply:
@vibhor98 Thanks. Tomorrow I upload the changes.

Reply:
Done